code/game/objects/items/wayfinding.dm

```diff
@@ -2,18 +2,45 @@
+	desc = "A machine given the thankless job of trying to sell wayfinding pinpointers. They point to common locations."
 	density = FALSE
 	layer = HIGH_OBJ_LAYER
+	armor = list(MELEE = 80, BULLET = 30, LASER = 30, ENERGY = 60, BOMB = 90, BIO = 0, RAD = 0, FIRE = 100, ACID = 80)
+	///List of user-specific cooldowns to prevent pinpointer spam.
 	var/list/user_spawn_cooldowns = list()
+	///List of user-specific cooldowns to prevent message spam.
 	var/list/user_interact_cooldowns = list()
-	var/spawn_cooldown = 5 MINUTES //time per person to spawn another pinpointer
-	var/interact_cooldown = 20 SECONDS //time per person for subsequent interactions
-	var/start_bal = 200 //how much money it starts with to cover wayfinder refunds
-	var/refund_amt = 40 //how much money recycling a pinpointer rewards you
+	///Time per person to spawn another pinpointer.
+	var/spawn_cooldown = 3 MINUTES
+	///Time per person for subsequent interactions.
+	var/interact_cooldown = 6 SECONDS
+	///How many credits the dispenser account starts with to cover wayfinder refunds.
+	var/start_bal = 400
+	///How many credits recycling a pinpointer rewards you.
+	var/refund_amt = 40
 	var/datum/bank_account/synth_acc = new /datum/bank_account/remote
-	var/ppt_cost = 65 //Jan 6 '20: Assistant can buy one roundstart (125 cr starting)
+	var/ppt_cost = 0 //Jan 9 '21: 2560 had its difficulties for NT as well
 	var/expression_timer
+	///Avoid being Reddit.
+	var/funnyprob = 1
+	///List of slogans used by the dispenser to attract customers.
+	var/list/slogan_list = list("Find a wayfinding pinpointer? Give it to me. I'll make it worth your while. Please. Daddy needs his medicine.", //the last sentence is a reference to Sealab 2021
+								"See a wayfinding pinpointer? Don't let it go to the crusher! Feed it to me instead! Please. I'll pay you.", //I see these things heading for disposals through cargo all the time
+								"Bleeding to death? Can't read? Find your way to medbay today!", //there are signs that point to medbay but you need basic literacy to get the most out of them
+								"Voted tenth best pinpointer in the universe in 2560!", //there were no more than ten pinpointers in the game in 2020
+								"Helping assistants find the departments they tide since 2560.", //not really but it's advertising
+								"These pinpointers are flying out the airlock!", //because they're being thrown into space
+								"Grey pinpointers for the grey tide!", //I didn't pick the colour but it works
+								"Feeling lost? Find direction in life with a wayfinding pinpointer.",
+								"Automate your sense of direction. Buy a wayfinding pinpointer today!",
+								"Feed me a stray pinpointer.", //this is an American Psycho reference
+								"We need a slogan!") //this is a Liberal Crime Squad reference
+	///Number of last entry we said in our slogan list.
+	var/previous_slogan = 0
+	///Last world tick we said a slogan.
+	var/last_slogan = 0
```

Comment:
Could you use the cooldown defines?

Reply:
Not sure. I'll do what I think you mean and you can tell me if it's right or not.

Reply:
Check `cooldowns.dm` for the defines I mean.